rosdep/ruby.yaml

```diff
@@ -82,6 +82,13 @@ ruby:
     xenial: [ruby, ruby-dev]
     yakkety: [ruby, ruby-dev]
     zesty: [ruby, ruby-dev]
+ruby-dev:
+  arch: [ruby]
+  debian: [ruby, ruby-dev]
+  fedora: [ruby, ruby-devel, openssl-devel, rubygems]
+  gentoo: [dev-lang/ruby]
+  macports: [ruby]
+  ubuntu: [ruby, ruby-dev]
```

Comment:
If this is to be a future only dev version should we not include the full ruby dependency here?